### ISSUE TITLE
MLT not found when launching flowblade-trunk

### ISSUE THREAD

**Issue by yichuang-zh** [jliljebl/flowblade-forum#35]
After git clone the Development Repository Version followed this web:
https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/docs/CREATING_TRANSLATION.md
I run command './flowblade' in the clone directory, the terminal tips me:
`MLT not found, exiting... `
`ERROR: No module named 'mlt' ` 

I checked the dependencies here:
https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/docs/DEPENDENCIES.md
There are two packages related to mlt: python-mlt and libmlt-data. And, they are already installed in my system.

So, what can I do now?

ps:
There are already a version 1.10.0 and a flatpak version of flowblade installed in my system.

My OS: debian stretch 9.11/gnome

**Comment by jliljebl** [comment]
Flowblade was converted to Python 3 and 2.4 will be the first version running in Python 3. You need MLT 6.18 to run current repo Flowblade.

We do not offer support for install issues but we do have a guide that might help installing MLT bindings from latest available MLT code: https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/docs/creating_user_folder_bindings.md

**Comment by yichuang-zh** [comment]
I had tried to install MLT bindings with debian buster but failed. It's a bit complicated, I need time, maybe I have to install a ubuntu virtual machine to test.

Creating a locale translation(Chinese zh-cn) is not as simple as I thought.

**Comment by jliljebl** [comment]
Yeah, this 2.4 will a bit difficult to contribute to, but i believe that MLT 6.18 starts hitting distro repos quite soon and situation will improve.

**Comment by yichuang-zh** [comment]
This sounds like an exciting news. I am looking forward.

**Comment by alexmyczko** [comment]
I also have a user with it: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=994775
